dev/breeze/src/airflow_breeze/cache.py

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+from typing import Any, List, Tuple
+
+from airflow_breeze import global_constants
+from airflow_breeze.breeze import get_airflow_sources_root
+
+# This file is to read the cached data in .build folder
+BUILD_CACHE_DIR = os.path.join(get_airflow_sources_root, '.build')
+
+
+def read_from_cache_file(param_name: str):
```

Comment:
```suggestion
def read_from_cache_file(param_name: str) -> Optional[str]:
```

I think it is better to return None if the parameter is missing in cache. In some cases "" (empty string) might be a good value.

Reply:
Is there any reason that makes returning None better?  @potiuk